utilities/uc-verification-request.vbs

```diff
@@ -241,60 +222,54 @@ FOR uc_membs = 0 to Ubound(uc_members_array, 2) 'start at the zero person and go
     member_info = member_info & vbNewLine & "Please review deductions and withholdings for this individual. " & vbNewLine
 NEXT
 
-IF team_email_dropdown = "ES Support" THEN team_email = "HSPH.ES.DEED@Hennepin.us"
-IF team_email_dropdown = "HC in METs" THEN team_email = "James.Berka@Hennepin.us; diane.beauchamp@hennepin.us"
-IF team_email_dropdown = "1800 Chicago" THEN team_email = "Jennifer.Moses@Hennepin.us"
-
 IF cca_checkbox = CHECKED THEN member_info = "CCA Request" & vbNewLine & member_info
 IF other_checkbox = CHECKED and other_check_editbox <> "" THEN member_info = "Other Request: " & other_check_editbox & vbNewLine & member_info
 
 CALL find_user_name(the_person_running_the_script)' this is for the signature in the email'
 
-If developer_mode = TRUE THEN send_email = FALSE
-'Creating the email
-'Call create_outlook_email(email_recip, email_recip_CC, email_subject, email_body, email_attachmentsend_email)
-IF send_email = TRUE THEN Call create_outlook_email(team_email, "", "UC Request for Case #" & MAXIS_case_number, member_info & vbNewLine & vbNewLine & "Submitted By: " & vbNewLine & the_person_running_the_script, "", TRUE)   'will create email, will send.
+'Call create_outlook_email(email_recip, email_recip_CC, email_subject, email_body, email_attachment,send_email)
+IF send_email = TRUE THEN Call create_outlook_email("HSPH.ES.DEED@Hennepin.us", "", "UC Request for Case #" & MAXIS_case_number, member_info & vbNewLine & vbNewLine & "Submitted By: " & vbNewLine & the_person_running_the_script, "", True)   'will create email, will send.
```

Comment:
Idea - add a MSGBOX if send_email is false. So we can see what it would look like if we are in TRAINING. This is not an issue with the code, but will make it more supportive.